db.go

```diff
@@ -372,3 +373,84 @@ func updateReceiptDateDelivered(messageID, recipientID int) error {
 	`, messageID, recipientID)
 	return err
 }
+
+func createMessageGroup(groupName string, memberIds []int) (*MessageGroup, error) {
+	tx, err := db.Begin()
```

Comment:
Good use of _transaction_ I think.

By the way, should we make _transaction_ a default way of doing SQL? Are there any downsides to it?

Reply:
I think we should use it only when needed. Not sure about downsides though.